docker-compose-db.yaml

```diff
@@ -0,0 +1,27 @@
+---
+# The project Docker compose file for development.
+
+version: '2.3'
+
+services:
+  config: &db-config
+    environment:
+      - PGHOST=db
+      - PGHOST_SLAVE=db
+      - PGSSLMODE=prefer
+  geoportal: *db-config
+  # geoportal-advance: *db-config
+  tools: *db-config
+  alembic: *db-config
+  # alembic-advance: *db-config
+  # webpack_dev_server: *db-config
+
+  db:
+    extends:
+      file: docker-compose-lib.yaml
+      service: db
+    volumes:
+      - postgresql_data:/var/lib/postgresql/data
+
+volumes:
```

Comment:
Maybe we should move volumes before services, for consistency with `docker-compose.yaml`